configs/gpalloc.conf.ctmpl

```diff
@@ -27,4 +27,9 @@ liquibase {
   changelog = "org/broadinstitute/dsde/workbench/gpalloc/liquibase/changelog.xml"
   initWithLiquibase = true
 }
+
+swagger {
+  googleClientId = "806222273987-fne0mu7nnof7ifg6e0uj6kqsdmk0tck2.apps.googleusercontent.com"
+  realm = "broad-dsde-local"
+}
```

Comment:
@helgridly I can't imagine above is what we want but I didn't know what else to insert.